tests/test_image_driver.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python 
+''' VIC Image Driver testing '''
+
+import xarray as xr
+import numpy as np 
+import numpy.testing as npt 
+
+def assert_nan_equal(ds_domain, ds_output): 
+    """ 
+    test to see if nans in two data arrays are the same in order to check domain 
+    first dataarray is domain file, second dataarray is a variable in the output
+    dataset for image driver 
+    """
+
+    # get lat/lng dim names from domain file 
+    lng_name, lat_name = ds_domain.coords
+
+    match_vars = [lng_name, lat_name] 
```

Comment:
`coords` will not always unpack in the order you are asserting it will here. It actually doesn't matter so you can generalize this.